src/components/SiteHeader.vue

```diff
@@ -113,6 +108,23 @@
             <template #label-prepend>
               <MenuIcon :type="item.value" />
             </template>
+            <template #label-append>
+              <div
+                v-if="item.value === 'notification'"
+                :class="[
+                  'flex',
+                  'justify-center',
+                  'items-center',
+                  'bg-shade-gray',
+                  {'bg-danger':getNotificationCount},
+                  'text-white',
+                  'text-[6px]',
+                  'rounded-[50%]',
+                  'w-[24px]',
```

Comment:
Is the fixed width enough for the text contained? I think dynamic size is better, use `rounded-full` in that case